### PR TITLE
Refactor in readme.md

### PR DESCRIPTION
**Pull request by cagodoy**
- [X] Update README with new implemented endpoints (ca)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +96,68 @@ Central notification service FTW.
...
 ```
 
+## Get message by id
+```
+  GET /api/v1/messages/:id
+```
+
+### Headers
+| Name | Type | Description |
+|-|-|-|-|
```
Comment (sebastianvera) [inline README.md:106]:
```suggestion
|-|-|-|
```

File: README.md [hunk 2]
```diff
@@ -90,6 +96,68 @@ Central notification service FTW.
...
+```
+
+## Cancel message
+```
+  POST /api/v1/messages/:id/cancel
+```
+
+### Headers
+| Name | Type | Description |
+|-|-|-|-|
```
Comment (sebastianvera) [inline README.md:149]:
```suggestion
|-|-|-|
```

**Review by sebastianvera — CHANGES_REQUESTED** [review]
The tables format is wrong, you have a typo

**Comment by mtavano** [comment]
You notice that you can't be more rat, when you submit a Readme's update PR, and you have a typo 🐀

**Pull request closed without merging**